Documentation/CMakeLists.txt

```diff
@@ -0,0 +1,132 @@
+cmake_minimum_required(VERSION 3.14)
+
+project("ElementDocumentation")
+
+# Add the cmake folder so the FindSphinx module is found
+set(CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/cmake" ${CMAKE_MODULE_PATH})
+
+
+## --------------------- Gather Project Files for Doxygen ---------------------
+
+set(SPHINX_RST
+        "index.rst"
+        "Overview.rst"
+        "UserGuide.rst"
+        "element_dotnet/index.rst"
+        "element_dotnet/Overview.rst"
+        "libelement/index.rst"
+        "libelement/Overview.rst")
+
+file(GLOB_RECURSE ELEMENT_DOTNET_SOURCES
+        ${CMAKE_SOURCE_DIR}/../Element.NET/*.cs)
+
+foreach (file IN LISTS ELEMENT_DOTNET_SOURCES)
+    string(APPEND ELEMENT_DOTNET_FILES " \"${file}\"")
+endforeach()
+
+file(GLOB_RECURSE LIBELEMENT_SOURCES
+        ${CMAKE_SOURCE_DIR}/../libelement/src/*.cpp
+        ${CMAKE_SOURCE_DIR}/../libelement/src/*.hpp
```

Comment:
could the /src/ globs be removed? only the libelement API (in include) matters for user documentation :)

Reply:
Sure thing, removed them 👍